### ISSUE TITLE
Postman not launching

### ISSUE THREAD

**Issue by OsamaNabih** [a85/POSTMan-Chrome-Extension#4382]
I had installed Postman previously and used it once. When I tried opening it again today, it didn't launch. I tried several times (tried running as admin too), but nothing happened. It existed in my system tasks but looked idle
![pm](https://user-images.githubusercontent.com/22377258/37893368-112e505c-30db-11e8-9f0e-502b46408649.jpg)
I uninstalled it and went over to the website and downloaded it again (version 6.0.10), when launching the setup I got a "installing postman" but nothing launching. It installed and I tried running it again and got the same issue, nothing happening.

**Comment by SamvelRaja** [comment]
@OsamaNabih  can you try opening the app using `cmd`?
You can open it from `C:\\Users\<USERNAME>\AppData\Local\app-6.0.10\Postman.exe` and send us the logs from the cmd window.
If still fails, kindly backup, `C:\\Users\<USERNAME>\AppData\Roaming\Postman` and delete the old one.[This will delete all postman data if it is existed before]

**Comment by OsamaNabih** [comment]
![pm2](https://user-images.githubusercontent.com/22377258/37894554-dfccb04a-30de-11e8-9bac-e81e62c663a1.jpg)
Am I doing this properly? This is all I get.

I deleted the Roaming/Postman folder and re-launched, same thing.

**Comment by SamvelRaja** [comment]
@OsamaNabih  this is odd. can you provide your OS and system information?

**Comment by OsamaNabih** [comment]
Windows 10
![pm3](https://user-images.githubusercontent.com/22377258/37903788-82311758-30f9-11e8-8c69-81950efccc67.jpg)
I'm working with the chrome extension for now, since the app one isn't working for me.

**Comment by SamvelRaja** [comment]
@OsamaNabih  can you open the `cmd` as admin and then open the app via that?
Also make sure no Postman process is running in your task manager before opening the app.

**Comment by vmans** [comment]
I have the exact same issue here. Difference: it never opened, not the first time either.

"The more software you use the more time you lose" so I leave it by that.

**Comment by stianlagstad** [comment]
Related to https://github.com/postmanlabs/postman-app-support/issues/4359, maybe?

**Comment by charleskey510** [comment]
I have the same issue with Windows 10 (64-bit).  I'm a first-time user.  I have reinstalled twice and Postman has failed to launch 3 times.  It spin-locks on initialization screen "Preparing your workspaces...This may take a few minutes."

In addition to this thread, this bug has appeared in #4359 as well and hasn't been resolved as far as I can see.

When started from command prompt in Administrator mode, get the following dump>>

c:\Users\Charles\AppData\Local\Postman\app-6.0.10>postman

c:\Users\Charles\AppData\Local\Postman\app-6.0.10>
ORM initialization failed:  { Error: EPERM: operation not permitted, rename 'C:\Users\Charles\AppData\Roaming\Postman\window~' -> 'C:\Users\Charles\AppData\Roaming\Postman\window'
  errno: -4048,
  code: 'EPERM',
  syscall: 'rename',
  path: 'C:\\Users\\Charles\\AppData\\Roaming\\Postman\\window~',
  dest: 'C:\\Users\\Charles\\AppData\\Roaming\\Postman\\window' }
Uncaught errors TypeError: Cannot read property 'collections' of undefined
    at bootstrapModels (c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\main.js:60:25)
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\bootstrap-models.js:26:14
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\waterline\lib\waterline.js:918:25
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\waterline\lib\waterline.js:719:25
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\waterline\node_modules\async\dist\async.js:952:25
    at iteratorCallback (c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\waterline\node_modules\async\dist\async.js:995:17)
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\waterline\node_modules\async\dist\async.js:847:20
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\async\lib\async.js:52:16
    at done (c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\async\lib\async.js:241:17)
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\async\lib\async.js:44:16
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\common\adapters\waterline-shared-nedb.js:320:29
    at newArguments.(anonymous function) (c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\nedb\lib\executor.js:29:17)
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\nedb\lib\persistence.js:305:26
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\nedb\lib\persistence.js:138:23
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\nedb\lib\storage.js:105:30
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\nedb\node_modules\async\lib\async.js:485:30
    at c:\Users\Charles\AppData\Local\Postman\app-6.0.10\resources\app.asar\node_modules\nedb\lib\storage.js:102:70
    at FSReqWrap.oncomplete (fs.js:112:15)

**Comment by RickHough** [comment]
On my Windows 8.1 box I just installed 6.1.3. Install completed but on clicking the launch icon I see a flashing blue cursor (timer?) and nothing else happens. Has anyone managed to get Postman 6.1.3 working on Windows 8.1 yet?

**Comment by asalhani** [comment]
@SamvelRaja 
I have the same version after I upgraded Postman to version 6.1.3. 
When I open Postman app, the app window keeps flashing and nothing appears.
I tried to run it through cmd (with and without admin. priviliage) but same thing.
As per @SamvelRaja  suggestion, I switched to a single monitor mode, but it didn't work either.
Find log below:

![postman issue](https://user-images.githubusercontent.com/5410830/41524749-289360a4-72e7-11e8-9d29-a0315ee12591.JPG)

```
C:\Users\Adeeb\AppData\Local\Postman\app-6.1.3>
ORM initialization successful
App events bus { name: 'booted', namespace: 'shared' }
App updater event { name: 'checkForElectronVersionUpdated',
  namespace: 'appUpdate' }
App events bus { name: 'booted', namespace: 'requester' }
[64732:0618/110156.645:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.646:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.649:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.649:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.650:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.651:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.651:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.652:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.652:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.652:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.653:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.654:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.654:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.655:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.655:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.655:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.657:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.658:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.658:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.659:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.659:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.660:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.660:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.661:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.661:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.662:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.662:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.663:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
[64732:0618/110156.663:ERROR:gles2_cmd_decoder.cc(17652)] [.DisplayCompositor-00
0000FCDC550CA0]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIU
M: invalid mailbox name
[64732:0618/110156.664:ERROR:gles2_cmd_decoder.cc(9709)] [.DisplayCompositor-000
000FCDC550CA0]RENDER WARNING: texture bound to texture unit 0 is not renderable.
 It maybe non-power-of-2 and have incompatible texture filtering.
```

**Comment by sivcan** [comment]
@asalhani @RickHough  Your issue seems to be similar to the one mentioned in https://github.com/postmanlabs/postman-app-support/issues/4594 . Can you try this as mentioned in the comment https://github.com/postmanlabs/postman-app-support/issues/4594#issuecomment-391601621

**Comment by asalhani** [comment]
@sivcan @RickHough 
I can confirm that [the environmental variable](https://github.com/postmanlabs/postman-app-support/issues/4594#issuecomment-391601621) is working in 6.1.3.
Thanks for the support :+1:

**Comment by dasre21** [comment]
[-1529685306363.log](https://github.com/postmanlabs/postman-app-support/files/2128451/-1529685306363.log)
[-1529685332068.log](https://github.com/postmanlabs/postman-app-support/files/2128452/-1529685332068.log)
Postman is not launching in my laptop. Any solution?

**Comment by SamvelRaja** [comment]
@dasre21  From the logs, we suspect that the initial load of data from the DB was corrupted. 
If you are already logged in user, I request you to delete the  PostmanData.
Linux: ~/.config/Postman
OSX - ~/Library/Application \Support/Postman
Windows: c:\users\[USER]\AppData\Roaming\Postman

⚠️  This will delete your existing local postman data from your machine.

**Comment by SamvelRaja** [comment]
@OsamaNabih  This issue still persists in our 6.1.4 app? Kindly confirm it.

**Comment by bruniasty** [comment]
6.1.4 now working (Win 8.1).  The main window keeps flashing and nothing works.

![image](https://user-images.githubusercontent.com/1955268/42058757-aa1b1cc4-7b21-11e8-9841-6625e63f0941.png)

**Comment by SamvelRaja** [comment]
@bruniasty  This seems to the GPU issue mentioned by @sivcan  Kindly try the solution mentioned in https://github.com/postmanlabs/postman-app-support/issues/4594#issuecomment-391601621 and let us know.

**Comment by SamvelRaja** [comment]
Closing this on inactivity on the original problem faced by the creator.

**Comment by mdvdhurk** [comment]
Ran into the same issue (Server 2008R2). Ran the application in compatibility mode for Windows 8, now it starts fine!

**Comment by rhexcasas** [comment]
Not running on Windows 8.1.

**Comment by danikane** [comment]
Postman will not start if NODE_OPTIONS environment variable is present in Windows. Took me a while to realize that. 😄 

https://github.com/electron/electron/issues/12695

**Comment by Vasilis-7** [comment]
I had the same issue.
For the last 3 days i couldn't launch Postman.
I removed the folder postman from roaming folder, in my desktop folder.
Postman opened, but it was empty.
I closed it, and i copied from my desktop's folder, postman, the folders databases, IndexedDB, Local Storage and Postman_Config and replaced with them the corresponding folders in the new postman folder in roaming.
I launched Postman with success and my **saved** data where there.
